Formula/frei0r.rb

```diff
@@ -13,10 +14,13 @@ class Frei0r < Formula
   end
 
   depends_on "autoconf" => :build
+  depends_on "pkg-config" => :build
+  depends_on "cairo" => :optional
+  depends_on "homebrew/science/opencv" => :optional
 
   def install
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
+    ENV["CAIRO_CFLAGS"] = "-I#{Formula["cairo"].opt_include}/cairo" if build.with? "cairo"
```

Comment:
Yes, CC can't find out <cairo.h> without `CAIRO_CFLAGS` because it wont see /cairo directory.